config/initializers/rack_rewrite.rb

```diff
@@ -0,0 +1,55 @@
+# Rack::Rewrite rules
+#
+# Includes redirects for old portal query params to Rails/Blacklight equivalents
+Rails.application.configure do
+  config.middleware.insert_before(Rack::Runtime, Rack::Rewrite) do
+    # Redirect searches with old params to app's equivalents
+    # @todo Move the logic into a class
+    r301 %r{/search(.html)?(\?.*)},
+      lambda { |match, rack_env|
```

Comment:
Unused block argument - `match`. If it's necessary, use `_` or `_match` as an argument name to indicate that it won't be used.<br>Align the parameters of a method call if they span more than one line.<br>Avoid using `{...}` for multi-line blocks.